.github/workflows/target-testing.yml

```
@@ -26,16 +26,15 @@ jobs:
         run: |
           sudo apt-get update -y
           sudo apt-get install -y 
-          sudo apt-get remove -y llvm-12
+          sudo apt-get remove -y llvm-12 llvm-14 llvm-11 clang-11 clang-format-11 clang-tidy-11 clang-format-14 clang-tidy-14 
```

FYI: Consider re-ordering by version (one more case) below. You also probably need to remove `clang-12`, `clang-format-12`? 
```suggestion
          sudo apt-get remove -y llvm-14 clang-format-14 clang-tidy-14 llvm-12 llvm-11 clang-11 clang-format-11 clang-tidy-11 
```